packages/frontend/src/utils/token.ts

```diff
@@ -0,0 +1,44 @@
+export const BRIDGED_CONSTANT = 'Bridged';
+
+interface ISpecialTokenMeta {
+    symbol: string;
+}
+
+const specialTokenMetaMap: { [contractName: string]: ISpecialTokenMeta } = {
```

Comment:
Could actually be named `specialFTMetadataMap` to correspond with how its being used.

This kind of static data should also generally be in a different file, keep the actual utility file as clean as possible.